src/App.js

```diff
@@ -1,12 +1,19 @@
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Home from './components/Home/Home';
+import Countries from './components/Countries/Countries';
 
 function App() {
   return (
-<section>
-  <h1>
-    CAPSTONE
-  </h1>
-</section>
+    <section className="container wrapper">
+
+      <Routes>
+        <Route exact path="/" element={<Home />} />
+
+        <Route path="/countries" element={<Countries />} />
```

Comment:
- [ ] Each country should have its own unique route. I'd suggest taking a look at the `react-router-dom` dynamic routing.